src/Processors/QtiV2/Out/QuestionTypes/McqMapper.php

```diff
@@ -9,15 +9,21 @@
 use LearnosityQti\Entities\QuestionTypes\mcq_options_item;
 use LearnosityQti\Services\LogService;
 use LearnosityQti\Utils\QtiMarshallerUtil;
+use qtism\common\utils\Version;
 use qtism\common\utils\Format;
 use qtism\data\content\FlowStaticCollection;
+use qtism\data\content\FeedbackInline;
+use qtism\data\content\InlineCollection;
+use qtism\data\content\SimpleInline;
```

Comment:
Is this used anywhere?

Reply:
I have removed all unused use statements.